core/src/main/java/bisq/core/dao/burningman/accounting/node/full/AccountingBlockParser.java

```diff
@@ -0,0 +1,210 @@
+    // The DPT can be very narrowly detected. The BTC fee txs might have false positives.
+    private Optional<AccountingTx> toAccountingTx(TempAccountingTx tempAccountingTx,
+                                                  Map<String, String> burningManNameByAddress,
+                                                  String genesisTxId) {
+        if (genesisTxId.equals(tempAccountingTx.getTxId())) {
+            return Optional.empty();
+        }
+
+        Set<String> receiverAddresses = burningManNameByAddress.keySet();
+        // DPT has 1 input from P2WSH with lock time and sequence number set.
+        // We only use native segwit P2SH, so we expect a txInWitness
+        List<TempAccountingTxInput> inputs = tempAccountingTx.getInputs();
+        List<TempAccountingTxOutput> outputs = tempAccountingTx.getOutputs();
+        // Max DPT output amount is currently 4 BTC. Max. security deposit is 50% of trade amount.
+        // We give some extra headroom to cover potential future changes.
+        // We store value as integer in protobuf to safe space, so max. possible value is 21.47483647 BTC.
+        long maxDptValue = 6 * Coin.COIN.getValue();
+        if (inputs.size() == 1 &&
+                isValidTimeLock(tempAccountingTx, inputs.get(0)) &&
+                doAllOutputsMatchReceivers(outputs, receiverAddresses) &&
+                isWitnessDataWP2SH(inputs.get(0).getTxInWitness()) &&
+                outputs.stream().allMatch(txOutput -> isExpectedScriptType(txOutput, tempAccountingTx)) &&
+                outputs.stream().allMatch(txOutput -> txOutput.getValue() <= maxDptValue)) {
+
+            List<AccountingTxOutput> accountingTxOutputs = outputs.stream()
+                    .map(output -> new AccountingTxOutput(output.getValue(), burningManNameByAddress.get(output.getAddress())))
+                    .collect(Collectors.toList());
+            return Optional.of(new AccountingTx(AccountingTx.Type.DPT_TX, accountingTxOutputs, tempAccountingTx.getTxId()));
+        }
+
+        // BTC trade fee tx has 2 or 3 outputs.
+        // First output to receiver, second reserved for trade and an optional 3rd for change.
+        // Address check is done in parse method above already.
+        // Amounts are in a certain range but as fee can change with DAO param changes we cannot set hard limits.
+        // The min. trade fee in Nov. 2022 is 5000 sat. We use 2500 as lower bound.
+        // The largest taker fee for a 2 BTC trade is about 0.0059976 BTC (599_760 sat). We use 1_000_000 sat as upper bound.
+        // Inputs are not constrained.
+        // Max fees amount is currently 0.0176 BTC.
+        // We give some extra headroom to cover potential future fee changes.
+        // We store value as integer in protobuf to safe space, so max. possible value is 21.47483647 BTC.
+        long maxTradeFeeValue = Coin.parseCoin("0.1").getValue();
```

Comment:
The comment says 1_000_000 sats but this is 10_000_000 sats. I think 10m is correct and the comment should be updated.

Reply:
Ok, will fix